Source/WebCore/page/Quirks.cpp

```diff
@@ -1394,113 +1096,92 @@ Quirks::StorageAccessResult Quirks::triggerOptionalStorageAccessQuirk(Element& e
 // youtube.com rdar://66242343
 bool Quirks::needsVP9FullRangeFlagQuirk() const
 {
-    if (!needsQuirks())
-        return false;
-
-    if (!m_quirksData.needsVP9FullRangeFlagQuirk)
-        m_quirksData.needsVP9FullRangeFlagQuirk = m_document->url().host() == "www.youtube.com"_s;
-
-    return *m_quirksData.needsVP9FullRangeFlagQuirk;
+    return needsQuirks() && m_quirksData.needsVP9FullRangeFlagQuirk;
 }
 
+// facebook.com: rdar://67273166
+// forbes.com:
+// reddit.com: rdar://80550715
+// twitter.com: rdar://73369869
 bool Quirks::requiresUserGestureToPauseInPictureInPicture() const
 {
 #if ENABLE(VIDEO_PRESENTATION_MODE)
     // Facebook, X (twitter), and Reddit will naively pause a <video> element that has scrolled out of the viewport,
     // regardless of whether that element is currently in PiP mode.
     // We should remove the quirk once <rdar://problem/67273166>, <rdar://problem/73369869>, and <rdar://problem/80645747> have been fixed.
-    if (!needsQuirks())
-        return false;
-
-    if (!m_quirksData.requiresUserGestureToPauseInPictureInPictureQuirk) {
-        auto domain = RegistrableDomain(topDocumentURL()).string();
-        m_quirksData.requiresUserGestureToPauseInPictureInPictureQuirk = isFacebook() || isDomain("x.com"_s) || isDomain("reddit.com"_s) || isDomain("forbes.com"_s);
-    }
-
-    return *m_quirksData.requiresUserGestureToPauseInPictureInPictureQuirk;
+    return needsQuirks() && m_quirksData.requiresUserGestureToPauseInPictureInPictureQuirk;
 #else
     return false;
 #endif
 }
 
+// bbc.co.uk rdar://126494734
```

Comment:
Just for consistency!
```suggestion
// bbc.co.uk: rdar://126494734
```